app/views/users/profile.html.erb

```diff
@@ -69,9 +69,17 @@
 
   <div>
     <i class="fas fa-user-friends"></i>
-    <a href="/people/<%= @profile_user.username %>/followers" class="link-secondary"><%= pluralize(@profile_user.followers.count, 'follower', plural: 'followers')%></a>
-    &#8226;
-    <a href="/people/<%= @profile_user.username %>/following" class="link-secondary"><%= pluralize(@profile_user.following_users.count, 'following', plural: 'following')%></a>
+    <% if @profile_user.followers.count>0 %>
+      <a href="/people/<%= @profile_user.username %>/followers" class="link-secondary"><%= pluralize(@profile_user.followers.count, 'follower', plural: 'followers')%></a>
+      &#8226;
```

Comment:
Line 74 should be on 78. Apart from that everything looks good. Great job!

Reply:
> Line 74 should be on 78. Apart from that everything looks good. Great job!

Thanks! I've done the change.